README.md

```diff
@@ -353,10 +364,10 @@ types offer a configuration that goes beyond that of the basic smell options, fo
 [Data Clump](docs/Data-Clump.md).
 All options that go beyond the [Basic Smell Options](docs/Basic-Smell-Options.md)
 are documented in the corresponding smell type /docs page (if you want to get a quick overview over all possible
-configurations you can also check out [the `default.reek` file in this repository](defaults.reek).
+configurations you can also check out [the `default.reek.yml` file in this repository](defaults.reek.yml).
```

Comment:
Let's use 'defaults' in the link text as well.

Reply:
Ah, good catch!